components/card/LazyCards.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="lazyScroll">
```

Comment:
This could probably be moved to the main Card component with a `lazy` prop that sets the class.

Also, for consistency, it should probably be `lazy-scroll`.

Reply:
My idea with this component was to prepare for the further work that the approach mentioned in #274 requires. I thought that, although both can get merged at some point, as an starting point was better to have a wrapper that does all the heavy lifting for us so pages that doesn't require Dom recycling doesn't need major changes.

Also, my first idea of this was to swap the cards with the ``skeleton-card`` component while scrolling fast (so it made more sense to have an actual component, unlike the current pushed code), but I wasn't able to get it working in a way I liked as I couldn't access the ``window.isScrollingFast`` variable like I mentioned earlier.

Reply:
> my first idea of this was to swap the cards with the skeleton-card component while scrolling fast 

This would generate more repaints + component mounting/unmounting/patching, which could degrade perf probably (The point of the DOM recycling is to avoid as much reflow as possible, which the different components would definitely lead to).